13_factorial/README.md

```diff
@@ -0,0 +1,12 @@
+# Exercise 13 - Factorial
+
+Write a recursive function that takes a non-negative integer, and returns the product of all positive integers less than or equal to the input integer. An input of `0` should return `1`. The function should only accept numbers, so `'4'` should not be accepted as it is a string. All invalid inputs should return `undefined`.
```

Comment:
How do we feel about having a link in here? Maybe to something like https://simple.wikipedia.org/wiki/Factorial

Reply:
Agree with this